exe/stewfinder

```diff
@@ -3,4 +3,11 @@ require 'stewfinder'
 
 filename = ARGV[0] ? File.absolute_path(ARGV[0]) : Dir.pwd
 
-Stewfinder.new(filename).find
+stewards = Stewfinder.new(filename).find
+
+puts 'Stewards for this file:'
```

Comment:
This script should really only have one responsibility -- passing arguments into the gem.

If you'd like to make this a method in stewfinder you can but it doesn't belong here.